static/scss/audio-player.scss

```diff
@@ -0,0 +1,269 @@
+
+      @include breakpoint(desktop) {
+        width: 36%;
+        max-width: 490px;
+      }
+      @media only screen and (max-width: 960px) and (min-width: 800px) {
+        width: 25%;
+      }
+      @include breakpoint(mobile) {
+        text-align: center;
+        width: 100%;
+      }
+
+      .audio-player-titles {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        height: min-content;
+
+        @include breakpoint(desktop) {
+          padding: 0px 12px;
+        }
+
+        @include breakpoint(mobile) {
+          display: flex;
+          flex-direction: column-reverse;
+        }
+
+        .audio-player-title {
+          font-size: 16px;
+          color: $font-grey-light;
+          max-height: 25px;
+
+          @include breakpoint(desktop) {
+            font-size: 16px;
+            margin-bottom: 10px;
+          }
+          @include breakpoint(mobile) {
+            font-size: 12px;
+          }
+        }
+
+        .audio-player-description {
+          display: block;
+          font-weight: bold;
+          font-size: 18px;
+          overflow: hidden;
+          text-overflow: ellipsis;
+          white-space: nowrap;
+          max-height: 46px;
+
+          @include breakpoint(desktop) {
+            font-size: 18px;
+          }
+          @include breakpoint(mobile) {
+            font-size: 14px;
+          }
+        }
+      }
+    }
+
+    .audio-player-controls {
+      display: flex;
+      flex: 1;
+      justify-content: left;
+      align-items: center;
+
+      @include breakpoint(desktop) {
+        flex-direction: row;
+        border-left: 1px solid rgba(0, 0, 0, 0.1);
+        padding-left: 16px;
+        height: 100%;
+      }
+      @include breakpoint(mobile) {
+        flex-direction: column;
+        width: 100%;
+      }
+
+      .audio-player-button {
+        height: 40px;
+        display: flex;
+        align-items: center;
+        padding-left: 12px;
+        padding-right: 12px;
+      }
+
+      .audio-player-playback-controls {
+        display: flex;
+        align-items: center;
+
+        @include breakpoint(desktop) {
+          width: 180px;
+        }
+
+        @include breakpoint(mobile) {
+          width: 100%;
+        }
+
+        .audio-player-button-container {
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          width: 100%;
+
+          .audio-player-button {
+            display: inline-block;
+            color: $rouge;
+            height: auto;
+          }
+
+          .audio-player-button span,
+          progress {
+            cursor: pointer;
+          }
+
+          .amplitude-play-pause span {
+            font-size: 48px;
+          }
+
+          .amplitude-play-pause.amplitude-playing span:before {
+            content: "pause_circle_outline";
+          }
+
+          .amplitude-play-pause.amplitude-paused span:before {
+            content: "play_circle_outline";
+          }
```

Comment:
is it necessary to change the display of the icon this way because it's laggy otherwise or something? just wondering if it would be easier to grab the boolean for whether it's playing or not out of `Amplitude` and use that to switch between these two icons in the react component itself

Reply:
I switched this up so that AudioPlayer.js uses Amplitude's `initialized` callback to set event handlers on the `Audio` element's `play` and `pause` events.  I liked the CSS solution because it was technically less code and doesn't require any JS.  However I like the JS solution more now because I realized that instead of programmatically clicking the button, we can use the `Amplitude.play()` function and simply run `setAudioPlaying(true)` to set the state.